go.mod

```diff
@@ -1,3 +1,8 @@
 module github.com/goreapers/gograveyard
 
 go 1.19
+
+require (
+	github.com/davecgh/go-spew v1.1.1
```

Comment:
I don't see this used anywhere. Is there a larger change that uses this coming?

Reply:
Oops, good catch no I was just using that for debugging